test/Integration/GetWorks.ts

```diff
@@ -0,0 +1,96 @@
+/* tslint:disable:no-relative-imports */
```

Comment:
I wasn't sure that I could test for a certain number of claims since it is a changing number.

Reply:
I was hard-coding an expected amount of works in some cases because the database is being prepared for tests. See [test/PrepareDB.ts](https://github.com/poetapp/node/blob/master/test/PrepareDB.ts).

I don't know what the best approach is since we don't want individual tests to depend on the order they run or other tests running first, but I guess we can at least rely on the database being prepared... so you can assume the number of works should be equal or greater than the prepared amount.

It would be nice to have a test that checks that the output from `GET /works` differs from `GET /works?publicKey=...`.

Reply:
Ok I've added a test that checks that `GET /works` returns more claims than `GET /works?publicKey=...` and a test to ensure that `GET /works` returns at least 3 claims.